content.md

```diff
@@ -1,23 +1,278 @@
+
+To install the `code` command, type Cmd+Shift+P and start typing `install code` and select "Install 'code' command in PATH". This will add the `code` command to your shell so you can open up VS Code from your terminal.
+
+![CODE path screenshot](./assets/code_path.png)
+
+<aside>The PATH is an environment variable in operating systems like Unix, Linux, and macOS, as well as in Windows. It specifies a set of directories where executable programs are located. When you type a command in the command line interface (CLI), the operating system searches through these directories specified in the PATH variable to find the executable corresponding to that command.
+
+For example, if you type `ls` in the terminal, the operating system will search through the directories listed in the PATH variable to find the `ls` executable, which is typically located in `/bin` or `/usr/bin`.
+
+The PATH variable helps the operating system locate executable files, making it easier to run commands from the command line without having to specify the full path to the executable every time.</aside>
+
+```bash
+# Examples
+code myfile.txt # open a specific file
+code myfolder # open a specific folder
+```
+
+## Setup a Package Manager
+
+### What is Homebrew?
+
+Homebrew is a package manager for macOS (and Linux) that simplifies the process of installing, updating, and managing software packages and libraries. It's like an app store for your command line. Here's what it does:
+
+1. **Package Installation**: Homebrew allows users to easily install software packages and libraries from the command line. Instead of searching for installation files or downloading from websites manually, Homebrew handles everything for you.
+2. **Dependency Management**: Many software packages rely on other libraries or tools to function properly. Homebrew automatically manages these dependencies, ensuring that everything required for a package to work is installed correctly.
+3. **Updates**: Homebrew simplifies the process of keeping software up to date. Users can easily update all installed packages with a single command, ensuring that they have the latest features and security patches.
+4. **Version Control**: Homebrew allows users to install specific versions of software packages if needed. This can be useful for compatibility reasons or if a user prefers an older version for some reason.
+5. **Configuration Management**: Homebrew provides options for configuring how packages are installed and managed, giving users flexibility and control over their software environment.
+
+Overall, Homebrew streamlines the process of managing software on macOS, making it easier for users to install, update, and maintain the tools they need for development and everyday use.
+
```

Comment:
I don't think we need this "heres what it does" list. The intro is good.

Reply:
Agree